app/models/contributor.rb

```diff
@@ -10,13 +10,15 @@ class Contributor < ApplicationRecord
   has_many :received_requests, -> { reorder(created_at: :desc).distinct }, source: :request, through: :received_messages
 
   has_one_attached :avatar
+  has_one :json_web_token, dependent: :destroy
+  accepts_nested_attributes_for :json_web_token
 
   acts_as_taggable_on :tags
 
   default_scope { order(:first_name, :last_name) }
   scope :active, -> { where(deactivated_at: nil) }
 
-  validates :data_processing_consent, acceptance: true, on: :contributor_signup
+  validates :data_processing_consent, acceptance: true, on: :create
```

Comment:
This introduces a new bug (related to the bug we initially introduced when adding validation for the data processing consent). This time, you won’t be able to manually create a contributor via the UI.

Why can’t we keep the previous `contributor_signup` context?

Reply:
I'll check this as soon as I have time. From the law perspective I'd say we should validate `data_processing_context` when personal data is added to our system. The validation shouldn't be called explicitly, but by default instead.

Reply:
> From the law perspective I'd say we should validate data_processing_context when personal data is added to our system. The validation shouldn't be called explicitly, but by default instead.

Yes, that’s fine for me. Then we’d need to make sure we explicitly skip the data processing validation when manually creating contributors.

Reply:
I added a hidden input when `contributor.new_record?` in 5ac3f6101a2e57e399d86720ce5982bc7c277237

Reply:
Speaking of 5ac3f6101a2e57e399d86720ce5982bc7c277237 I removed `status: bad_request` when validations fail. It's rails default behaviour to send 200 on data validations and show error messages to the user.